extensions/firefox/content/PdfjsChromeUtils.jsm

```diff
@@ -190,16 +189,16 @@ let PdfjsChromeUtils = {
       findPrevious: aEvent.detail.findPrevious
     };
 
-    let chromeWindow = aEvent.target.ownerDocument.defaultView;
-    let browser = chromeWindow.gBrowser.selectedBrowser;
-    if (this._browsers.has(browser)) {
-      // Only forward the events if the selected browser is a registered
-      // browser.
-      let mm = browser.messageManager;
-      mm.sendAsyncMessage('PDFJS:Child:handleEvent',
-                          { type: type, detail: detail });
-      aEvent.preventDefault();
+    let browser = aEvent.currentTarget.browser;
```

Comment:
*Note:* As far as I can tell, `aEvent.target.browser` works equally well. But the comment *specially* asked for `currentTarget`, so that's what I used.